content/en/docs/Advanced/bundles.md

```diff
@@ -67,6 +67,12 @@ metallb:
   address_pool: 192.168.1.10-192.168.1.20
 ```
 
+{{% alert title="Warning" %}}
+When a bundle is installed after Kairos installation (see above), changing the
+bundle's configuration block after Kairos installation is complete,
+will not have any effect.
+{{% /alert %}}
```

Comment:
I'm not sure about this one. We need to try this out with both types of bundle installation: https://kairos.io/docs/advanced/bundles/#consuming-bundles and verify. E.g. Can I change the configuration for a bundle that gets installed before Kubernetes starts at any time? What does it mean "before Kubernetes starts" when I'm using a core image?